apps/nextra/pages/en/build/smart-contracts/digital-asset.mdx

```diff
@@ -2,416 +2,321 @@
 
-Collection and token addresses will also be used to query data such as fetching
-all resources via fullnode API or with an indexing service.
+1. Tokens can be easily extended since they are implemented using Move [Objects](objects.mdx).
+2. You can now directly transfer NFTs without the recipient “opting-in” on-chain.
+3. NFTs can own other NFTs for easy composability. 
 
-### Royalties
+If you want a simple way to mint NFTs without the ability to customize or extend their functionality, you can use the `aptos_token` module which implements the DA standard (see the section on how to use it below). 
 
-Following the object extensibility pattern, royalties are added to collections
-or tokens as a resource with associated functionality provided by
-[the royalty module](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token-objects/sources/royalty.move)
-Royalty can be updated as long as a `MutatorRef`, a storable struct that grants
-permissions, is generated at creation time and stored.
+<Callout type="info">
+Note that all Digital Asset modules are deployed at the reserved address `0x4`.
+</Callout>
 
-See [Aptos Token](#aptos-token) for examples on how Royalty's `MutatorRef` can
-be stored and used. Royalty can also be set directly on a token if it has a
-different royalty config than the collection's.
+## Using the Digital Asset Standard
 
-## Token lifecycle
+This standard is implemented with two Objects:
 
-All Digital Asset modules are deployed at the reserved framework address `0x4`.
+1. `Collection`s - A set of NFTs with a name and a bit of context for the group.
+2. `Token`s - Digital assets which represent unique assets. These are often used to represent NFTs and usually use a `uri` link to point to more info about the asset (ex. a link to an image, video, etc.).
 
-### Collection creation
+<ThemedImage
+alt="Digital Asset token and collection relationship"
+sources={{
+    light: '/docs/digital-asset-light.svg',
+    dark: '/docs/digital-asset-dark.svg',
+  }}
+/>
 
-Every token belongs to a collection. The developer first needs to create a
-collection with:
+All `Token`s are required to have a reference to a parent `Collection`, but the parent `Collection` does **not** own the `Token`. Newly minted `Token`s are usually owned by the creator. From there, they can be transferred to other accounts.
 
-1. A fixed maximum supply. Current supply is tracked and cannot exceed the
-maximum set.
+### Creating a `Collection`
 
-```move
-module 0x42::example {
-  use aptos_token_objects::collection;
-  use std::string::utf8;
+There are two ways to create a `Collection` depending on whether you want there to be a maximum supply of NFTs it can hold:
 
-  public entry fun create_collection(creator: &signer) {
-    let max_supply = 1000;
+1. For a fixed maximum supply you can use `collection::create_fixed_collection`:
```

Comment:
```suggestion
### Fixed Maximum Supply

The maximum supply of the collection is fixed to a set number less than __MAX_BOUNDS__ and cannot be changed in the future. Doing this requires `collection::create_fixed_collection`
```

Reply:
I'm wary of including __MAX_BOUNDS__ without indicating where to find that value or what the number is. Do you happen to know what it is? (I couldn't find it from searching around). 

Updated the header and text otherwise :)